internal/adapters/dataproviders/rootstock/abi.go

```diff
@@ -0,0 +1,51 @@
+package rootstock
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/rsksmart/liquidity-provider-server/internal/adapters/dataproviders/rootstock/bindings"
+)
+
+type FlyoverABIs struct {
+	PegIn                *abi.ABI
+	PegOut               *abi.ABI
+	Discovery            *abi.ABI
+	CollateralManagement *abi.ABI
+	DaoContributor       *abi.ABI
+	Flyover              *abi.ABI
+}
+
+func MustLoadFlyoverABIs() *FlyoverABIs {
+	pegInAbi, err := bindings.IPegInMetaData.GetAbi()
+	if err != nil {
+		panic("could not load PegIn ABI: " + err.Error())
+	}
+	pegOutAbi, err := bindings.IPegOutMetaData.GetAbi()
+	if err != nil {
+		panic("could not load PegOut ABI: " + err.Error())
+	}
+	discoveryAbi, err := bindings.IFlyoverDiscoveryMetaData.GetAbi()
+	if err != nil {
+		panic("could not load Discovery ABI: " + err.Error())
+	}
+	collateralManagementAbi, err := bindings.IPegOutMetaData.GetAbi()
```

Comment:
I think there's an error here. The abi is getting obtained from the IPegOutMetaData. I think it should be
bindings.ICollateralManagementMetaData.GetAbi()
This is probably a copy paste error